helm/app-operator-chart/templates/rbac.yaml

```diff
@@ -0,0 +1,110 @@
+apiVersion: rbac.authorization.k8s.io/v1beta1
+kind: ClusterRole
+metadata:
+  name: {{ .Values.name }}
+  labels:
+    app: {{ .Values.name }}
+    giantswarm.io/service-type: "managed"
+rules:
+- apiGroups:
+  - apiextensions.k8s.io
+  resources:
+  - customresourcedefinitions
+  verbs:
+  - "*"
+- apiGroups:
+    - application.giantswarm.io
+  resources:
+    - apps
+  verbs:
+    - get
+    - list
+    - patch
+    - update
+    - watch
+- apiGroups:
+    - core.giantswarm.io
+  resources:
+    - charts/status
+  verbs:
+    - create
+    - patch
+    - update
+- apiGroups:
```

Comment:
don't we also need get (and maybe list?) or is that included in the above rules of charts?

Reply:
I would actually use '*' here. The operators role is to manage chart CRs. This is what we do in cluster-operator.

```
  - apiGroups:
      - application.giantswarm.io
    resources:
      - charts
    verbs:
      - "*"
```

Reply:
this is resolved right?

Reply:
Yes, it's up here https://github.com/giantswarm/app-operator/blob/790ca67fdaa50439485100352e472671a7c4a3d9/helm/app-operator-chart/templates/rbac.yaml#L96-L97